app/static/src/app/components/options/EditParamSettings.vue

```diff
@@ -0,0 +1,173 @@
+                   <option v-for="param in paramNames" :key="param">{{param}}</option>
+                 </select>
+               </div>
+             </div>
+             <div class="row mt-2" id="edit-scale-type">
+               <div class="col-5">
+                 <label class="col-form-label">Scale type</label>
+               </div>
+               <div class="col-6">
+                 <select class="form-select" v-model="settingsInternal.scaleType">
+                   <option v-for="scale in scaleValues" :key="scale">{{scale}}</option>
+                 </select>
+               </div>
+               <div class="col-6">
+             </div>
+             </div>
+            <div class="row mt-2" id="edit-variation-type">
+              <div class="col-5">
+                <label class="col-form-label">Variation type</label>
+              </div>
+              <div class="col-6">
+                <select class="form-select" v-model="settingsInternal.variationType">
+                  <option v-for="variation in variationTypeValues" :key="variation">{{variation}}</option>
+                </select>
+              </div>
+            </div>
+            <div v-if="settingsInternal.variationType === 'Percentage'" class="row mt-2" id="edit-percent">
+              <div class="col-5">
+                <label class="col-form-label">Variation (%)</label>
+              </div>
+              <div class="col-6">
+                <numeric-input :value="settingsInternal.variationPercentage"
+                               @update="(n) => settingsInternal.variationPercentage = n"></numeric-input>
+              </div>
+            </div>
+            <template v-else>
+              <div class="row mt-2" id="edit-from">
+                <div class="col-5">
+                  <label class="col-form-label">From</label>
+                </div>
+                <div class="col-6">
+                  <numeric-input :value="settingsInternal.rangeFrom"
+                                 @update="(n) => settingsInternal.rangeFrom = n"></numeric-input>
+                </div>
+              </div>
+              <div class="row mt-2" id="edit-to">
+                <div class="col-5">
+                  <label class="col-form-label">To</label>
+                </div>
+                <div class="col-6">
+                  <numeric-input :value="settingsInternal.rangeTo"
+                                 @update="(n) => settingsInternal.rangeTo = n"></numeric-input>
+                </div>
+              </div>
+            </template>
+            <div class="row mt-2" id="edit-runs">
+              <div class="col-5">
+                <label class="col-form-label">Number of runs</label>
+              </div>
+              <div class="col-6">
+                <numeric-input :value="settingsInternal.numberOfRuns"
+                               @update="(n) => settingsInternal.numberOfRuns = n"></numeric-input>
```

Comment:
do you want to limit this here to some max? it matters much less now that everything is on the client really

Reply:
I think we said we'd limit this to 20, there's a separate ticket for that